### PR TITLE
Start community-provided dataset docs

### PR DESCRIPTION
**Pull request by sshleifer**
This is one I did to get the pseudo-labels updated. Not sure if it generalizes, but I figured I would write it down. It was pretty easy because all I had to do was make properly formatted directories and change URLs.

+ In slack @thomwolf  called it a `user-namespace` dataset, but the docs call it `community dataset`.
I think the first naming is clearer, but I didn't address that here.

+ I didn't add metadata, will try that.

### CODE REVIEW

File: datasets/wmt_t2t/wmt_utils.py [hunk 1]
```diff
@@ -385,10 +385,18 @@ def get_path(self, src):
     SubDataset(
         name="uncorpus_v1",
         target="en",
-        sources={"ru", "zh"},
+        sources={"ru"},
```
Comment (sshleifer) [inline datasets/wmt_t2t/wmt_utils.py:388]: accidental, will rm

Comment (sshleifer) [reply datasets/wmt_t2t/wmt_utils.py:388]: actually the delta is just wrong.

**Comment by julien-c** [comment]
can you also reference the `--organization` flag like in https://github.com/huggingface/transformers/blob/master/docs/source/model_sharing.rst#upload-your-model-with-the-cli ?

**Comment by lhoestq** [comment]
Not sure if the changes in `datasets/wmt_t2t/wmt_utils.py` are intentional.
If you want to add more configs to wmt, could you do it in a serapate PR ?

**Comment by sshleifer** [comment]
I don't think I changed wmt_utils I think github is wrong.

**Comment by sshleifer** [comment]
Trying new fork.

**Pull request closed without merging**